app/views/feedback.jade

```diff
@@ -8,7 +8,8 @@ html
 
     link(rel='stylesheet' href='styles/main.css')
     script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
-    script(type="text/javascript" src="/scripts/satisfaction.js")
+    script(type="text/javascript" src="https://code.jquery.com/jquery-2.2.3.min.js")
+    script(type="text/javascript" src="/scripts/satisfaction.min.js")
```

Comment:
there are things in the main script that break my page and vice versa, I have no good reason for sticking my scripts in the head other than it's a habit